### ISSUE TITLE
acmepipe: cannot parse diff: 1d0

### ISSUE THREAD

**Issue by omar-polo** [mkhl/cmd#1]
Hi!

Thanks a lot for acmepipe/autoacme/acmeeval: they made my life with acme a lot easier.

I noted that whenever I try to format a buffer (with autoacme) that changes the first line of the file, acmepipe fails with `cannot parse diff: 1d0`.  I've found that removing the check on [main.go:232](https://github.com/mkhl/cmd/blob/master/acme/acmepipe/main.go#L232) solve the issue (and doesn't seem to cause problems).  That may be, however, a bit too brutal. I'm still relatively new to the various utilities from plan9 (for instance, I am able to deduce what the output of `9 diff ...` means -- it looks a lot like ed/sam commands -- but I'm not sure). I'll try to investigate a bit more, but in the meantime any help is appreciated :)

### steps to reproduce
1. `autoacme ./autofmt`, where `autofmt` contains:
```
#!/usr/bin/env rc

switch ($1$2) {
case put*.sorted
        sort $target | acmepipe
}
```

2. create `file.sorted` with
```
zzz
aaa
```

3. saving that file triggers the error message (and the file now contains `zzz aaa zzz`)

**Comment by mkhl** [comment]
> Thanks a lot for acmepipe/autoacme/acmeeval: they made my life with acme a lot easier.

Thank you, I'm very glad to hear!

As for the bug: Nice catch! Turns out diff can produce 0 for either number range, and your fix (removing the check) seems like the correct thing to do. Would you mind submitting a pull request that removes it?

**Comment by omar-polo** [comment]
Sure!

**Comment by mkhl** [comment]
Closed by #2
